x-pack/auditbeat/module/system/socket/_meta/docs.asciidoc

```diff
@@ -2,5 +2,148 @@
 
 beta[]
 
-This is the `socket` dataset of the system module.
+This is the `socket` dataset of the system module. It allows to monitor network
+traffic to and from running processes. It's main features are:
 
+- Supports TCP and UDP sockets over IPv4 and IPv6.
+- Outputs per-flow bytes and packets counters.
+- Enriches the flows with https://www.elastic.co/guide/en/ecs/current/ecs-process.html[process]
+and https://www.elastic.co/guide/en/ecs/current/ecs-user.html[user] information.
+- Provides information similar to Packetbeat's flow monitoring with reduced CPU
+and memory usage.
```

Comment:
I feel like the question "why use this when we have Packetbeat" should be answered but at the same time I'm afraid of making too-bold claims. It has a lesser impact on CPU and memory (and room for improvement), but a more in-depth benchmark is necessary.